src/components/PurchaseOrder/CloseOrder/CloseOrderModal.js

```diff
@@ -51,11 +63,42 @@ class CloseOrderModal extends Component {
         >
           <FormattedMessage id="ui-orders.paneBlock.closeBtn" />
         </Button>
-        <CloseOrderModalForm
-          close={this.closeModal}
-          orderId={orderId}
+        <Modal
+          label={<FormattedMessage id="ui-orders.closeOrderModal.title" values={{ orderId }} />}
           open={this.state.openModal}
-        />
+        >
+          <Row>
+            <Col xs={12}>
+              <Select
+                placeHolder="---Choose the reason---"
+                dataOptions={reasons}
+                required
+                label={<FormattedMessage id="ui-orders.closeOrderModal.reason" />}
+                onChange={(e => this.setState({ values: { reason: e.target.value } }))}
+              />
+              <TextArea
+                label={<FormattedMessage id="ui-orders.closeOrderModal.notes" />}
+                onChange={(e => this.setState({ values: { note: e.target.value } }))}
+              />
+            </Col>
+          </Row>
+          <Button
+            buttonStyle="primary"
+            onClick={() => {
+              closeOrderSubmit(this.state.values);
+              this.closeModal();
+            }}
+            style={{ float: 'right' }}
```

Comment:
I don't think using float is a common practice in Folio. I'm pretty sure there is a possibility to reach the same goal in other way